src/logic/calculate.js

```diff
@@ -0,0 +1,36 @@
+import operate from './operate';
+
+const calculate = (data, name) => {
```

Comment:
I think you need to import `Calculate` function in `app.js` because you will need it in the next milestone.
![image](https://user-images.githubusercontent.com/31897434/108829836-f50b4e80-75c8-11eb-829b-6d9e2bbc460e.png)
[checkout Pathwright](https://microverse.pathwright.com/library/react-redux-v11/184438/path/step/106376018/)